handlers/handlers.go

```diff
@@ -19,30 +20,37 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 	w.WriteHeader(status)
 }
 
```

Comment:
Public funcs should have a go doc

Reply:
Ah this showed as changed because I added the new functions at the bottom and then removed the ReleaseSample ones making the diff a bit odd. have reordered them now so that the diff looks better and left the existing functions as they were. They are only sample endpoints that are going soon anyway. This will help with the conflicts with Giles' branch as well 👍